var/spack/repos/builtin/packages/py-zarr/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyZarr(PythonPackage):
+    """Zarr is a Python package providing an implementation of chunked,
+    compressed, N-dimensional arrays."""
+
+    homepage = "https://zarr.readthedocs.io"
+    url      = "https://pypi.io/packages/source/z/zarr/zarr-2.3.2.tar.gz"
+
+    version('2.3.2', sha256='c62d0158fb287151c978904935a177b3d2d318dea3057cfbeac8541915dfa105')
+
+    depends_on('python@3.5:',               type=('build', 'run'))
```

Comment:
Drop this line, older versions of Python are indeed supported and require additional dependencies you'll have to add.